storage-proofs-porep/src/stacked/vanilla/params.rs

```diff
@@ -838,6 +839,60 @@ impl<Tree: MerkleTreeTrait, G: Hasher> Clone for TemporaryAux<Tree, G> {
 }
 
 impl<Tree: MerkleTreeTrait, G: Hasher> TemporaryAux<Tree, G> {
+    pub fn new(sector_nodes: usize, num_layers: usize, cache_path: PathBuf) -> Self {
+        let labels = (1..=num_layers)
+            .map(|layer| StoreConfig {
+                path: cache_path.clone(),
+                id: CacheKey::label_layer(layer),
+                size: Some(sector_nodes),
+                // The TreeC doesn't discard any rows, hence it can be set to 0.
+                rows_to_discard: 0,
+            })
+            .collect();
+
+        let tree_d_size = get_merkle_tree_len(sector_nodes, BINARY_ARITY)
+            .expect("Tree must have enough leaves and have an arity of power of two");
+        let tree_d_config = StoreConfig {
+            path: cache_path.clone(),
+            id: CacheKey::CommDTree.to_string(),
+            size: Some(tree_d_size),
+            // The TreeD doesn't discard any rows, hence it can be set to 0.
+            rows_to_discard: 0,
+        };
+
+        let tree_size = get_merkle_tree_len(sector_nodes, Tree::Arity::to_usize())
+            .expect("Tree must have enough leaves and have an arity of power of two");
+
+        let tree_r_last_config = StoreConfig {
+            path: cache_path.clone(),
+            id: CacheKey::CommRLastTree.to_string(),
+            size: Some(tree_size),
+            // A default 'rows_to_discard' value will be chosen for tree_r_last, unless the user
```

Comment:
I think it's possible that this part/assumption may break things.  It's reasonable, but may require some more thought/testing across those cases.

Reply:
Why would it break things? This is copy and pasted from the original code. The only difference is that we take the default value/the env variable and don't read it from disk.

Is this a comment or blocking a merge? If it's blocking, what do you suggest?